src/test/ui/destructuring-assignment/tuple_struct_destructure_fail.stderr

```diff
@@ -31,6 +31,11 @@ LL | struct TupleStruct<S, T>(S, T);
 ...
 LL |     TupleStruct(_) = TupleStruct(1, 2);
    |     ^^^^^^^^^^^^^^ expected 2 fields, found 1
+   |
+help: use `_` to explicitly ignore each field
+   |
+LL |     TupleStruct(_, _) = TupleStruct(1, 2);
+   |                  ^^^
```

Comment:
These two cases should also be suggesting `..` as an alternative, no?

Reply:
Well, if someone is already explicitly listing out wildcards, should we suggest that they use `..`? On the other hand, someone may not know that `..` is an option and would rather that over listing out wildcards manually. So I think overall I agree with you that we should be giving it as an alternative :)

I'll change it, but for other things we can always adjust the behavior later; I'm hoping to have this PR finished soon.

Reply:
Sounds reasonable, I don't foresee any big changes. I think that the code itself looks good but want to nail down the exact behavior we want.

Reply:
So is the behavior you're suggesting that we also suggest `S(..)` if it's currently of the form `S(_, _, _)`?

Reply:
Yes, I think that if it is currently `S(_, _)` and we suggest `S(_, _, _)` we should also suggest `S(..)`. Having said that, if you find that it will be too much work, only create a ticket and r=me.